twx

```diff
@@ -208,11 +254,21 @@ import_ext() {
     local input_params="$1"
     local url="$TWX_URL/ExtensionPackageUploader?purpose=import&validate=false"
 
-    local http_status=$(
-        curl -X POST "$http_option" -o "response-import-ext.json" -w '%{http_code}' -F "file=@$input_params" \
-            -H "appKey:$TWX_APPKEY" \
-            -H "X-XSRF-TOKEN:TWX-XSRF-TOKEN-VALUE" "$url"
-    )
+    local http_status
+
+    if [ $use_appkey -eq "1" ]; then
+        http_status=$(
+            MSYS_NO_PATHCONV=1 curl -X POST "$http_option" -o "response-import-ext.json" -w '%{http_code}' -F "file=@$input_params" \
```

Comment:
Just thought -- can we rename those files to something like `tmp-response-import-ext.json`, just to highlight their temporary nature?

Reply:
Done